docs/topics/snippet_jdk-hardware-mac-prerequisites.adoc

```diff
@@ -1,11 +1,8 @@
 :_content-type: SNIPPET
```

Comment:
@HagayVider1 @anarnold97 
Are you sure that this snippet should be changed? Now it is used for installation of IntelliJ, VS code extension, web console and CLI (but not Eclipse). Do the changes in prerequisites apply to those components? If they do, you can use the same snippet in Eclipse instead of the Prerequisites section.

Reply:
@sbeskin-redhat I have reverted the snippets

@HagayVider1 - speak to Phil as soon as possible

Reply:
@HagayVider1 the QE team tested against the 2023-06 version of Eclipse. So I agree with that change.

Reply:
@HagayVider1 - as Sasha says, we use this snippet in other docbooks, remove the snippet `include` for Eclipse and add what you had directly, i.e:

* Java Development Kit (JDK) 17 is installed and applied as the default. {ProductShortName} supports the following JDKs:

** OpenJDK 17
** Oracle JDK 17
** Eclipse Temurin™ JDK 17

@sbeskin-redhat - is that ok with you?

Reply:
@HagayVider1 - I have dealt with this